src/sentry/incidents/subscription_processor.py

```diff
@@ -129,7 +129,7 @@ def process_update(self, subscription_update):
 
         aggregation = QueryAggregations(self.alert_rule.aggregation)
         aggregation_name = query_aggregation_to_snuba[aggregation][2]
-        aggregation_value = subscription_update["values"]["data"][aggregation_name]
+        aggregation_value = subscription_update["values"]["data"][0][aggregation_name]
```

Comment:
It might be a good idea to assert (or at least log a warning) on the length here if it's not 1.

Reply:
I'll do that as a followup so we can test this change out.